README.md

```diff
@@ -1 +1,10 @@
-# skilld-docker-nginx
\ No newline at end of file
+# skilld-docker-nginx
+
+## Environment Variables
```

Comment:
actually it is not true, because this vars applied only once at container build time